sites/cnfpt/src/backend/locale/fr/LC_MESSAGES/django.po

```diff
@@ -21,32 +21,52 @@ msgstr ""
 msgid "Unsorted uploads are not allowed."
 msgstr "Les téléchargements non classés sont interdits"
 
-#: cnfpt/settings.py:210
+#: cnfpt/settings.py:213
+msgid "Dashboard"
+msgstr ""
```

Comment:
translate to "Tableau de bord"?

By the way I'm not sure why these changes pop-up and only in fr language?

Reply:
Because for CNFPT this is the only one language and there is just an offset of this translation.